src/reactionsystem.jl

```diff
@@ -544,7 +544,7 @@ struct ReactionSystem{V <: NetworkProperties} <:
     # inner constructor is considered private and may change between non-breaking releases.
     function ReactionSystem(eqs, rxs, iv, sivs, unknowns, spcs, ps, var_to_name, observed,
                             name, systems, defaults, connection_type, nps, cls, cevs, devs,
-                            metadata = nothing, complete::Bool = false; checks::Bool = true)
+                            metadata, complete; checks::Bool = true)
```

Comment:
Not fully sure how you want this function, but this is basically the base constructor from the args, so did like I understood it and our conversation from Wednesday.